ios/FluentUI/Button/ButtonLegacy.swift

```diff
@@ -12,14 +12,22 @@ import Combine
 @IBDesignable
 @objc(MSFButtonLegacy)
 open class MSFButtonLegacy: UIButton, TokenizedControlInternal {
```

Comment:
because we have change direction, i think this should be the default, and lets rename the swiftui one.

Reply:
Also agree, but will be changed in a different PR.